src/utils/compute-title.js

```diff
@@ -8,6 +8,16 @@
 const MAX_TITLE_LENGTH = 106;
 const SEPARATOR = ' | ';
 
+export const computeFullPath = (parents) => {
+    let testedPath = '';
+    let path = '';
```

Comment:
'path' variable seems not necessary

Reply:
One of the variables is indeed useless, I removed testedPath and kept path for semantics reasons